vhdbuilder/packer/packer_source.sh

```diff
@@ -247,6 +247,8 @@ copyPackerFiles() {
     SSHD_CONFIG_SRC=/home/packer/sshd_config_1604
   elif [[ ${UBUNTU_RELEASE} == "18.04" && ${ENABLE_FIPS,,} == "true" ]]; then
     SSHD_CONFIG_SRC=/home/packer/sshd_config_1804_fips
+  elif [[ ${UBUNTU_RELEASE} == "22.04" && ${ENABLE_FIPS,,} == "true" ]]; then
+    SSHD_CONFIG_SRC=/home/packer/sshd_config_2204_fips
```

Comment:
nit: i know it might be out of scope of this PR, but I would really like to consolidate these files to being a common file and then appending distro specific bits to it to avoid redundant code.

Reply:
sure that's fair, I can look into templating the file in the future